cmd/kubelet/app/server.go

```diff
@@ -1338,7 +1342,8 @@ func createAndInitKubelet(kubeServer *options.KubeletServer,
 		kubeServer.RegisterSchedulable,
 		kubeServer.NodeLabels,
 		kubeServer.NodeStatusMaxImages,
-		kubeServer.KubeletFlags.SeccompDefault || kubeServer.KubeletConfiguration.SeccompDefault)
+		kubeServer.KubeletFlags.SeccompDefault || kubeServer.KubeletConfiguration.SeccompDefault,
+		kubeServer.Flagz)
```

Comment:
nit: Can we add `Flagz` to the kubedeps (kubelet.Dependencies) instead?

Reply:
I fully agree with your suggestion